docs/integrations/cli.md

```diff
@@ -41,6 +41,10 @@ You can also install the standalone package:
   ```
 :::
 
+::: info
+If you are not able to find the binary (e.g. with `pnpm` and only `unocss` is installed), you'll need to explicit install `@unocss/cli` standalone package.
+:::
```

Comment:
We could probably also remove L16-28

Reply:
If I remove L16-28, then the tip I added makes no sense, since we are letting user explicitly install it anyway. Which approach do you prefer?

- Remove L16-28, remove what I've added, and Remove `you can also install the standalone package:`
Or
- Keep it as is.

Reply:
Let's remove L16-28 then. It won't work anyway

Reply:
No, it works with `npm`, or `yarn`, and I believe (guess) with `pnpm`'s `--shamefully-hoist` being enabled.